docs/source/usersguide/depletion.rst

```diff
@@ -51,3 +51,87 @@ time::
 
 Note that the coupling between the transport solver and the transmutation solver
 happens in-memory rather than by reading/writing files on disk.
+
+Caveats
+=======
+
+Energy Deposition
+-----------------
+
+The default energy deposition mode, ``"fission-q"``, instructs the
+:class:`openmc.deplete.Operator` to normalize reaction rates using the product
+of fission reaction rates and fission Q values taken from the depletion chain.
+This approach does not consider indirect contributions to energy deposition,
+including neutron heating or kinetic energy of fission fragments. In doing this,
```

Comment:
kinetic energy of fission fragments is most definitely included (it's the bulk of fission energy!). The important things we miss are neutron heating and heating due to secondary photons from non-fission reactions.